packages/ui/cypress/component/STabsPanel.spec.cy.ts

```diff
@@ -0,0 +1,175 @@
+import { mount } from '@cypress/vue'
+import { config } from '@vue/test-utils'
+import { STabsPanel, STab, useTabsStateApi, TabsPanelApi } from '@/lib'
+
+before(() => {
+  config.global.components = { STabsPanel, STab }
+})
+
+after(() => {
+  config.global.components = {}
+})
+
+describe('STabsPanel', () => {
+  const getEl = (id: string) => cy.get(`[data-cy=${id}]`)
+
+  it('The first and the last tabs in Panel have border-radius', () => {
+    mount({
+      template: `
+      <STabsPanel>
+        <STab :name="'left'" data-cy="left">left</STab>
+        <STab :name="'center'" data-cy="center">center</STab>
+        <STab :name="'right'" data-cy="right">right</STab>
+      </STabsPanel>
+      `,
+    })
+    getEl('left').should('have.css', 'border-radius', '8px 0px 0px 8px')
+    getEl('center').should('have.css', 'border-radius', '0px')
+    getEl('right').should('have.css', 'border-radius', '0px 8px 8px 0px')
+  })
+
+  it('Disabled prop works correctly', () => {
+    mount({
+      template: `
+        <STabsPanel>
+          <STab :disabled="true" data-cy="disabled">disabled</STab>
+          <STab data-cy="notDisabled">notDisabled</STab>
+        </STabsPanel>
+      `,
+    })
+    getEl('disabled').should('be.disabled')
+    getEl('notDisabled').should('not.be.disabled')
+  })
+
+  it('Background type prop is primary', () => {
+    mount({
+      template: `
+        <STabsPanel :background="'primary'" data-cy="primary">
+          <STab data-cy="tabName" :name="tabName">TabTitle</STab>
+        </STabsPanel>
+      `,
+    })
+    getEl('tabName').should('have.class', 's-tab_background-primary')
+  })
+
+  it('Background type prop is secondary', () => {
+    mount({
+      template: `
+        <STabsPanel :background="'secondary'" data-cy="secondary">
+          <STab data-cy="tabName" :name="tabName">TabTitle</STab>
+        </STabsPanel>
+      `,
+    })
+    getEl('tabName').should('have.class', 's-tab_background-secondary')
+  })
+
+  it('Background type prop is none', () => {

```

Comment:
"When background prop on STabsPanel is set to none, then STab background is set accordingly" would be more clear here.